### DISCUSSION TITLE
如何自定义input 图标

### DISCUSSION THREAD

**Discussion by hihuangwei** [ant-design/ant-design#31612] [Q&A]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

![QQ浏览器截图20210731141434](https://user-images.githubusercontent.com/3166799/127757364-8d3737f2-c166-48b9-bc26-d38ed99f1c82.png)

**Comment by xrkffgg** [comment]
Input.Search 不支持自定义。

你可以使用 `addonAfter` 来自定义

**Comment by hihuangwei** [comment]
addonAfter 无点击回调 建议增加 @afc163
